pkg/sql/logictest/tests/local-vec-off/local-vec-off_test: TestLogic_udf failed

pkg/sql/logictest/tests/local-vec-off/local-vec-off_test.TestLogic_udf [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5905556?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/5905556?buildTab=artifacts#/) on master @ [1b77518e5194972df726380d12d247df5e7995b3](https://github.com/cockroachdb/cockroach/commits/1b77518e5194972df726380d12d247df5e7995b3):


```
=== RUN   TestLogic_udf
    test_log_scope.go:162: test logs captured to: /artifacts/tmp/_tmp/937c109841397bf9baf5846210863a8f/logTestLogic_udf53067594
    test_log_scope.go:80: use -show-logs to present logs inline
[08:13:53] rng seed: -2182730375082689945

    logic.go:2216: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2896/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/udf:18: 
        expected:
        pq: unimplemented: functions do not currently support \* expressions\nHINT: You have attempted to use a feature that is not yet implemented\.\nSee: https://go\.crdb\.dev/issue-v/10028/v22\.2
        got:
        pq: unimplemented: functions do not currently support * expressions
        HINT: You have attempted to use a feature that is not yet implemented.
        See: https://go.crdb.dev/issue-v/10028/dev
[08:13:53] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2896/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/udf with config local-vec-off: 6 tests, 1 failures
    logic.go:3570: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2896/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/udf:21: error while processing
    logic.go:3570: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2896/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-dbg/bin/pkg/sql/logictest/tests/local-vec-off/local-vec-off_test_/local-vec-off_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/udf:21: too many errors encountered, skipping the rest of the input
    panic.go:500: -- test log scope end --
--- FAIL: TestLogic_udf (0.61s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_udf.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-18170

--- a/pkg/sql/logictest/testdata/logic_test/udf
+++ b/pkg/sql/logictest/testdata/logic_test/udf
@@ -15,7 +15,7 @@ CREATE TABLE t_implicit_type(a INT PRIMARY KEY, b STRING);
 statement error pq: return type mismatch in function declared to return int\nDETAIL: Actual return type is record
 CREATE FUNCTION f() RETURNS INT IMMUTABLE LANGUAGE SQL AS $$ SELECT a, b from t_implicit_type $$
 
-statement error pq: unimplemented: functions do not currently support \* expressions\nHINT: You have attempted to use a feature that is not yet implemented\.\nSee: https://go\.crdb\.dev/issue-v/10028/v22\.2
+statement error pq: unimplemented: functions do not currently support \* expressions.*
 CREATE FUNCTION f() RETURNS t_implicit_type IMMUTABLE LANGUAGE SQL AS $$ SELECT * from t_implicit_type $$
 
 # statement ok

